.eslintrc.js

```diff
@@ -55,6 +53,12 @@ module.exports = {
       files: [ '**/*.ts' ],
 
       parser: '@typescript-eslint/parser',
+      plugins: ['@typescript-eslint'],
+      extends: [
+        'plugin:@typescript-eslint/eslint-recommended',
+        'plugin:import/typescript',
+        'prettier/@typescript-eslint'
+      ],
```

Comment:
[`plugin:@typescript-eslint/eslint-recommended`](https://github.com/typescript-eslint/typescript-eslint/blob/v2.33.0/packages/eslint-plugin/src/configs/eslint-recommended.ts) turns off all the `eslint` rules that conflict with `@typescript-eslint`

[`plugin:import/typescript`](https://github.com/benmosher/eslint-plugin-import#typescript) [includes configuration](https://github.com/benmosher/eslint-plugin-import/blob/v2.20.2/config/typescript.js) when using `eslint-plugin-import` with TypeScript

`prettier/@typescript-eslint` turns off the `@typescript-eslint` rules that conflict with `prettier`